packages/common/src/index.ts

```diff
@@ -304,17 +304,12 @@ export const getFilterRuleWithDefaultValue = <T extends FilterRule>(
                         [TimeFrames.WEEK]: moment(
                             valueIsDate ? value : undefined,
                         )
-                            .utc(true)
                             .startOf('week')
                             .toDate(),
-                        [TimeFrames.MONTH]: moment()
-                            .utc(true)
```

Comment:
By forcing .utc(true), we were selecting a day before depending on the timezone when converting it back to Date()